sdk/csharp/libraries/microsoft.bot.solutions/Authentication/MultiProviderAuthDialog.cs

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.Bot.Solutions.Authentication
     /// </summary>
     public class MultiProviderAuthDialog : ComponentDialog
     {
+        private static readonly string[] _acceptedLocales = new string[] { "en", "de", "es", "fr", "it", "zh" };
```

Comment:
```suggestion
        private static readonly string[] acceptedLocales = new string[] { "en", "de", "es", "fr", "it", "zh" };
```

`_` it's a C# stuff, right?

Reply:
It kind of is, it is used for private variables.
It's just a naming convention that not everyone uses, but since I saw other private variables using it, I did the same.

Reply:
I'd like to rename the `_acceptedLocales` to `acceptedLocales` 🙏🏾 . I think that the other places that you mentioned it's because they have getter/setters